web/src/main/java/com/marklogic/hub/web/web/CurrentProjectController.java

```diff
@@ -300,8 +300,5 @@ private void clearAuthenticationAttributes(HttpServletRequest request) {
     @Override
     public void onLogoutSuccess(HttpServletRequest request, HttpServletResponse response, Authentication authentication) throws IOException, ServletException {
         request.getSession().invalidate();
-        String pluginDir = hubConfig.getHubPluginsDir().toString();
-        watcherService.removeListener(this);
-        watcherService.unwatch(pluginDir);
```

Comment:
Shouldn't we unwatch flows, stedef, entities dir snd also removelistener ?

Reply:
@srinathgit add them.

Reply:
Yes we should also unwatch them to clear it.